tokens/governance/treasury-spend.md

```diff
@@ -1,97 +1,43 @@
-
-To get started, head to [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss://wss.api.moonbase.moonbeam.network#/treasury){target=\_blank} and take the following steps:
+## Submitting Your Idea to the Forum {: #submitting-your-idea-to-the-forum }
 
-1. Select the **Governance** heading
-2. Click on **Treasury**
-3. Click **Submit Proposal**
-
-![Treasury home](/images/tokens/governance/treasury-proposals/treasury-proposal-2.webp)
-
-Then, take the following steps:
-
-1. Select the account that will be the creator of the Treasury proposal. Make sure that the account selected here is sufficiently funded with your proposal bond
-2. Select the beneficiary of the proposal. This should correspond to the account indicated in your forum post
-3. Indicate the value of the spend proposal. The required bond is {{ networks.moonbeam.treasury.proposal_bond}}% of the proposal's value. The bond committed to a Treasury spend proposal is irrevocable
-4. Review each field for accuracy then press **Submit proposal** and sign the transaction
+!!! note
+    You'll need to have an account on the [Moonbeam Forum](https://forum.moonbeam.network/){target=\_blank} and be logged in before submitting a Treasury spend proposal.
 
-![Submitting a Treasury spend proposal](/images/tokens/governance/treasury-proposals/treasury-proposal-3.webp)
+As mentioned in the happy path above, the first step of a Treasury proposal is to submit the proposal to the [Moonbeam Forum](https://forum.moonbeam.network/c/governance/Treasury-proposals/8){target=\_blank}. To properly submit a Treasury spend request, take the following steps:
 
-After submitting your Treasury proposal, you can refresh Polkadot.js Apps and you should see your proposal listed. If you don't see your proposal appear here, your account may not have contained sufficient funds for the required bond.
+1. From the [**Governance** section](https://forum.moonbeam.network/c/governance/Treasury-proposals/8){target=\_blank}, click **New Topic**. By starting the topic in the **Governance** section, the proposal will come pre-filled with a template to ensure you cover all the necessary points 
+2. Provide a title for the proposal
+3. Enter the contents of the proposal, covering: the Title and Proposal Status, a brief Abstract, the Motivation, a Project Overview with Team Experience, the Rationale, any Key Terms, the Overall Cost, the Use of Treasury Funds, the technical Specifications, and the Steps to Implement
```

Comment:
These are all UI elements that users will see in pre-filled template, so UI elements should all be bold